README.md

```diff
@@ -1,32 +1,116 @@
+      </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#roadmap">Roadmap</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#built-with">Built With</a></li>
+    <li><a href="#contact">Contact</a></li>
+  </ol>
+</details>
+
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+`vhs` is a versatile tool for network traffic capture. `vhs` can be run as a command line tool or be deployed into your
+Kubernetes cluster as a sidecar, where it can capture traffic to and from your services. Captured traffic
+can be analyzed to produce live Prometheus metrics or saved for use in offline analysis, load testing, or whatever
+you can imagine!
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+The quickest way to see `vhs` in action is to use the development Docker image in this repo to run a demo.
+
+### Prerequisites
+The first step is to clone the VHS repository.
 
-Record and replay HTTP traffic
-
-# Development
+```
+git clone https://github.com/rename-this/vhs.git
+```
 
-The easiest way to run this is to use the docker image included in the repo.
+You will need a working Docker installation to successfully execute the following commands. You should be able to
+install Docker from your favorite package manager, or you can see the [Docker site](https://www.docker.com/get-started)
+for more details.
 
-To build the image (which installs some helpful tools and mounts the source in the container):
+Once you have Docker set up and working, you can build the development container (which installs some helpful tools
+and mounts the source in the container) by changing directory into the repository you cloned and running:
 
 ```
 $ make dev
 ```
 
-Open two bash sessions on the container. 
+### Demo Setup
+Open a bash session on the container by running the following command in a terminal:
 
-In one, run this script to boot a simple echo server and a curl request that calls the server every second:
+```
+$ docker exect -it vhs_dev bash
+```
+
+In your new bash session, run this script to start a simple echo server and a curl request that calls the server every
+second:
 
 ```
 $ cd testdata && ./echo.bash
 ```
 
-In the other, run the following command to build `vhs`, and run it against `0.0.0.0:1111` using a dummy middleware that appends `" [[hijacked <HTTP_MESSAGE_TYPE>]]"` to the end of the request or response body (note this command discards `stderr` because it can be noisy at times):
+This will generate some local HTTP traffic for `vhs` to capture.
+
+### Run The Demo
+Open another bash session on the container in a new terminal:
+
+```
+$ docker exect -it vhs_dev bash
+```
+
+In this new session, run the following command to build `vhs` and run it to capture our local HTTP traffic.
 
 ```
 $ go build ./cmd/vhs && ./vhs --input "tcp|http" --output "json|stdout" --capture-response --address 0.0.0.0:1111 --middleware ./testdata/http_middleware.bash | jq -R "fromjson | .body" 2> /dev/null
```

Comment:
fyI: prefixing commands with `$` normally (according to markdownlinter) means you show the full output of the command. If you just want to show the command as is, without any output.

See https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md014---dollar-signs-used-before-commands-without-showing-output

Reply:
Fixed in most recent commit.